docs/content/pages/tutorials/translation.md

```diff
@@ -32,3 +32,69 @@ Here's an example, for `russian.json`:
     }
 }
 ```
+
+# CLI Tool
```

Comment:
Markdown best practice dictates that there shouldn't be more than one top-level heading, you may want to make this an `h2` and increment all the levels of the below headings by one.

Reply:
Actually, I had my own concerns about that.
When I viewed the Ship log page, I saw that there were multiple top-level headings, so I did the same.
However, I will change it if it looks like it should be fixed.
Thanks for the review ::)